steam/manifest.py

```diff
@@ -924,6 +934,10 @@ def __init__(
         self._stats_visible = common.get("community_visible_stats", "0") == "1"
         self._free = extended.get("isfreeapp", "0") == "1"
 
+        self.content_descriptors = [
+            ContentDescriptor.try_value(int(id)) for id in common.get("content_descriptors", MultiDict()).values()
+        ]
```

Comment:
Please update the types.manifest file for this and add a small docstring for it

Reply:
done